caddyhttp/fastcgi/fastcgi.go

```diff
@@ -91,7 +91,14 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error)
 			fcgiBackend.SetSendTimeout(rule.SendTimeout)
 
 			var resp *http.Response
-			contentLength, _ := strconv.Atoi(r.Header.Get("Content-Length"))
+
+			var contentLength int64
```

Comment:
I chose int64 because quic-go(https://github.com/lucas-clemente/quic-go/blob/master/h2quic/request.go#L49) uses it. I thought it would be better to use the same type with quic-go for overflow safe.